### ISSUE TITLE
@Markup for variables?

### ISSUE THREAD

**Issue by misstricky** [holidaypirates/nucleus#33]
Hello! Loving Nucleus so far. I have some variables where it would be great to be able to show examples. Any chance this could ever be added in?

Thanks!!

**Comment by Rhym** [comment]
Could you give an example?

**Comment by misstricky** [comment]
Easiest for me to show in the non-nucleus version of the styleguide (trying to convert it to nucleus).

We've got font-size variables. It would be nice to show what these actually look like:
![image](https://cloud.githubusercontent.com/assets/353156/20408140/33f73906-ace2-11e6-88a2-5f228c584959.png)

Right now as I'm setting nucleus up this looks like:
![image](https://cloud.githubusercontent.com/assets/353156/20408177/4d449566-ace2-11e6-81b2-b3fe2ab64c38.png)

**Comment by Rhym** [comment]
I suppose a nuclide for font sizing similar to the colour nuclide could be added. I think for the interim I would apply these variables to a class e.g .base-font--small, and applying them in markup. An example of this:

```
/**
 * @atom Font Sizing
 * @section Typography
 * @markup
 *  <p class="base-font--xsmall">$xsmall-base-font</p>
 *  <p class="base-font--small">$small-base-font</p>
 *  ...
 */
.base-font {
  &--xsmall {
    font-size: $xsmall-base-font;
  }
  //...
}
```

**Comment by misstricky** [comment]
@Rhym yup I have those classes from making them for the non-automated old styleguide :) Yeah i was going to do that as an alternative to using the @nuclide even though they are technically variables -- thanks!
